cypress/integration/blocks-listing.js

```diff
@@ -425,7 +395,87 @@ if (Cypress.env('API') !== 'guillotina') {
         .contains('Document outside Folder')
         .should('not.exist');
     });
-    it('Listing block - Test Criteria: Location Navigation path', () => {
+    it('Listing block - Test Criteria: Location relative particular to a issue', () => {
+      cy.createContent({
```

Comment:
@iFlameing could you please add the lines from my BDD scenario (GIVEN, WHEN, THEN) as subheadlines to the test for better readability?

Reply:
onto it

Reply:
Done :)